CMakeLists.txt

```diff
@@ -54,6 +54,8 @@ if (WIN32)
 	add_subdirectory(updater)
 endif()
 
+add_subdirectory(pcsx2-gsrunner EXCLUDE_FROM_ALL)
```

Comment:
We check a flag `ENABLE_GSRUNNER` a little below in the same file
The warning it prints will need to be updated, however.

Reply:
The fun of leaving patches sitting around for years and then rebasing them

Should be fixed